examples/guestbook/README.md

```diff
@@ -67,7 +67,82 @@ The web front end interacts with the redis master via javascript redis API calls
 
 This example requires a running Kubernetes cluster.  See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
 
-### Step One: Start up the redis master
+### Step One: Fire up the redis master service
```

Comment:
I would like to see a Quick Start section, which just launched the application with the minimal number of commands. If the user is interested in a deep dive, then they could read the rest of the example.

This would:
* put all configuration in a single subdirectory
* put services and replication controllers together in the same files
* ideally create all the resources with one command: kubectl create -f subdir/

We should try putting all the resources in a single file, also, and just fetch it via http:
kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/guestbook/all-in-one.yaml

If https doesn't work, please file an issue.